test/provision/compile.sh

```diff
@@ -16,7 +16,7 @@ cd ${GOPATH}/src/github.com/cilium/cilium
 if echo $(hostname) | grep "k8s" -q;
 then
     if [[ "$(hostname)" == "k8s1" ]]; then
-        make docker-image
+        make LOCKDEBUG_ARG=1 docker-image
```

Comment:
I realise this is still wip, but happened to notice that this is doing a `make` with `LOCKDEBUG_ARG`, rather than `LOCKDEBUG` like the other `make` invocations do.

Reply:
^+1 I am trying to understand this as well. Why do we need `LOCKDEBUG_ARG` as well as `LOCKDEBUG`

Reply:
There is no need, will fix this in next upload.

Reply:
Done.